nugets/Devolutions.NowClient/Devolutions.NowClient.csproj

```diff
@@ -0,0 +1,14 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>net8.0</TargetFramework>
+    <ImplicitUsings>enable</ImplicitUsings>
+    <Nullable>enable</Nullable>
+    <Platforms>Win32;x64;ARM64</Platforms>
```

Comment:
question: No support for Linux/macOS? Is this relying on many platform specific stuff?

Reply:
No platform-specific stuff at all, I'll change Win32 to x86, accidentally selected it instead of x86 in solutions configurations via VisualStudio GUI, thanks for spotting!